### PR TITLE
 executor, infoschema: return a pseudo profiling table when system var profiling is set to 1

### LINKED ISSUE #5666: Better Compatibility With Using Navicat for MySQL As TiDB Client
When I execute SQL in Navicat, It always print an error behind my execute result。
 For example：
sql:
````
SELECT * from xxxx
````
I  Find This In tidb.log .Also ,this error would display in  Navicat too.
````
2018/01/17 11:33:00.363 metrics.go:355: [warning] [EXPENSIVE_QUERY] SELECT * from  xxxx
2018/01/17 11:33:00.513 session.go:702: [warning] [29] parse error:
[parser:1064]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '/*100,3), '%') AS `百分比` FROM INFORMATION_SCHEMA.PROFILING WHERE QUERY_ID= ' at line 1
SELECT STATE AS `状态`, ROUND(SUM(DURATION),7) AS `期间`, CONCAT(ROUND(SUM(DURATION)/*100,3), '%') AS `百分比` FROM INFORMATION_SCHEMA.PROFILING WHERE QUERY_ID= GROUP BY STATE ORDER BY SEQ

````
My Tidb Version:
````
release v1.0.3
````
My Navicat Version:
````
11.0.10
````

### PR DESCRIPTION
**Pull request by XuHuaiyu**
fix #5666 
Navicat autorun sql 
``` sql
set @@profiling = 1;
SELECT QUERY_ID, SUM(DURATION) AS SUM_DURATION FROM INFORMATION_SCHEMA.PROFILING GROUP BY QUERY_ID;
```
 to get the QUERY_ID.

This commit returns a **pseudo** profiling table when profiling is set to 1.

PTAL @coocood

### CODE REVIEW

File: infoschema/tables.go [hunk 1]
```diff
@@ -1033,6 +1060,9 @@ func (it *infoschemaTable) getRows(ctx sessionctx.Context, cols []*table.Column)
 		fullRows = dataForTableConstraints(dbs)
 	case tableFiles:
 	case tableProfiling:
+		if v, ok := ctx.GetSessionVars().GetSystemVar("profiling"); ok && v == "1" {
```
Comment (coocood) [inline infoschema/tables.go]: use `tidbOptOn`.

**Comment by XuHuaiyu** [comment]
PTAL @coocood

File: sessionctx/variable/varsutil.go [hunk 2]
```diff
@@ -123,8 +123,8 @@ func SetSessionSystemVar(vars *SessionVars, name string, value types.Datum) erro
 	return vars.SetSystemVar(name, sVal)
 }
 
-// tidbOptOn could be used for all tidb session variable options, we use "ON"/1 to turn on those options.
-func tidbOptOn(opt string) bool {
+// TIDBOptOn could be used for all tidb session variable options, we use "ON"/1 to turn on those options.
```
Comment (zimulala) [inline sessionctx/variable/varsutil.go]: s/TIDBOptOn/TiDBOptOn ?

**Comment by XuHuaiyu** [comment]
PTAL @coocood @zimulala

**Comment by coocood** [comment]
LGTM

**Pull request merged**